src/org/sosy_lab/java_smt/solvers/opensmt/OpenSmtEvaluator.java

```diff
@@ -0,0 +1,35 @@
+// This file is part of JavaSMT,
+// an API wrapper for a collection of SMT solvers:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2022 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.sosy_lab.java_smt.solvers.opensmt;
+
+import com.google.common.base.Preconditions;
+import opensmt.Logic;
+import opensmt.MainSolver;
+import opensmt.Model;
+import opensmt.PTRef;
+import opensmt.SRef;
+import org.sosy_lab.java_smt.basicimpl.AbstractEvaluator;
+
+public class OpenSmtEvaluator extends AbstractEvaluator<PTRef, SRef, Logic> {
+
+  private final MainSolver osmtSolver;
```

Comment:
Is the osmtSolver unused?

Reply:
or is the reference required to avoid some garbage collection? then please document this.